src/views/Home/components/About/AboutBox/AboutBox.scss

```diff
@@ -0,0 +1,47 @@
+@use "./stylesheets/responsiveness";
+@use "./stylesheets/typography";
+
+.about-box {
+    z-index: 1;
+    border-radius: 33px;
+    width: 80%;
+    min-height: 331px;
```

Comment:
We probably don't need this. It leaves a huge empty space on certain devices.

Reply:
Just to clarify, are you referring to just min-height or all these lines?

Reply:
Just min-height!